egs2/mls/asr2/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+kmeans_feature=wavlm_large/21  # use model_type/layer_index
+nclusters=1000
+
+src_lang=$(echo "${kmeans_feature}_full_km${nclusters}" | tr "/" "_")
+tgt_lang=en
```

Comment:
Since the target language is not multilingual anymore, you may consider change it to `multi` or `multilingual` (the tag is mostly serving for token naming etc, so please feel free to change)

Reply:
Applied in https://github.com/espnet/espnet/pull/5441/commits/91c2b06069b95492b9f6eee9ffb5e1cc84bdb896 Thanks 👍